src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -21,6 +24,12 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
     if (hostConfig) {
       this.adaptiveCard.hostConfig = new AdaptiveCardsAPI.HostConfig(hostConfig);
     }
+    telemetry.trackEvent(TAB_CLICK_EVENT, 
+     {
+       ComponentName: 'Adaptive Cards Tab',
+       QueryUrl: sampleQuery ? sampleQuery.sampleUrl : null,
```

Comment:
is there any user entered information such as userid or other ids

Reply:
Yes, I am working on a function to redact the PII.